### PR TITLE
Create Latex document and Makefile

### PR DESCRIPTION
**Pull request by jksf**
I started an official document meant to make a specification of SignHash project. I decided to use Latex as I want a formal look with professional typesetting and easy versioning with git.

### CODE REVIEW

**Comment by jksf** [comment]
Here is the generated document in case you have a problem creating it. 
[signhash-whitepaper.pdf](https://github.com/SignHash/signhash-whitepaper/files/1388922/signhash-whitepaper.pdf)

File: signhash-whitepaper.tex [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+\item ensure file integrity
+\item verify file authenticity
+\item uniquely identify the file
+\end{list}
+
+A file can be corrupted not only by network errors and software bugs.
+It can be modified in transmission or storage by untrusted party, to include malicious code.
+
+This risk is especially significant in the days of cryptocurrencies.
+Private keys are a frequent target of hackers, so users need to take care of their system security.
```
Comment (biern) [inline signhash-whitepaper.tex:71]: Exposing wallet private keys is one of plethora of risks when running untrusted / unverified code, I'm wondering if this use case should be stressed out so much here.

**Comment by biern** [comment]
A very good start, good job.

File: signhash-whitepaper.tex (hunk 1 above)
Comment (jksf) [reply signhash-whitepaper.tex:71]: Thanks for this comment. I will try to expand this topic in the document, so private keys are not the only example.

**Pull request merged**